src/pages/tokenVerify/TokenVerification.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
You should have the naming more consistent 
src/pages/tokenVerify/tokenVerify.js